src/core/FBXWrangler.cpp

```diff
@@ -4807,7 +4807,12 @@ NiCollisionObjectRef FBXWrangler::build_physics(FbxNode* rigid_body, set<pair<Fb
 	{
 		bhkCollisionObjectRef collision;
 		if (export_rig)
-			collision = new bhkBlendCollisionObject();
+		{
```

Comment:
Dunno what Heir Gain and Vel Gain mean, but all vanilla meshes seem to have them at zero and it seem to improve their dynamics.